preload/cls/IPM/Installer.cls

```diff
@@ -19,22 +19,27 @@ XData PM [ XMLNamespace = INSTALLER ]
 <Default Name="TRACKINGID" Value="UA-126752118-1" />
 
 <Namespace Name="${CURNS}" Code="${CURNSRoutineDB}" Data="${CURNSGlobalDB}" Create="no" Ensemble="0">
-  <Configuration>
-      <ClassMapping Package="%IPM" From="${CURNSRoutineDB}" />
-      <RoutineMapping Routines="%IPM.*" Type="ALL" From="${CURNSRoutineDB}" /> 
-  </Configuration>
+  <!-- Support use in a bootstrapped ^INSTALL context, detected by presence of %Log -->
+  <If Condition="($Data(%Log)#2)">
```

Comment:
Should this happen when %Log is not defined? Currently a regular install (not in ^INSTALL) would not create the mappings since %Log is not defined right?

Reply:
This is just plain wrong. Going to kill it.